roles/confluent.test/molecule/rbac-scram-custom-rhel/verify.yml

```
@@ -35,6 +38,17 @@
         property: confluent.balancer.enable
         expected_value: "true"
 
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/kafka/server.properties
+        property: config.providers.securepass.class
+        expected_value: io.confluent.kafka.security.config.provider.SecurePassConfigProvider
+
+    - name: Check confluent cli is installed
+      command: "{{confluent_cli_path}} version"
```

```suggestion
    - name: Check confluent cli is installed
      command: "/opt/confluent-cli/confluent/confluent version"
```

secrets protection is turned off on this scenario

use full path in tests, please confirm that is the correct path

---

Should I change scenario? `zookeeper-mtls` fit perfectly and has secrets protection enabled.

---

sure, that works!

---

actually it would be nice (whichever scenario you choose) to test that the cli is on all hosts not just the kafka hosts, you could add a new play with `hosts: all`

---

moved tests to `zookeeper-mtls` and implemented `hosts: all`